### DISCUSSION TITLE
Class attributes and Class Instances autocompletion

### DISCUSSION THREAD

**Discussion by kavi6024** [microsoft/pylance-release#2654] [Enhancement]
Issue Type: <b>Feature Request</b>
Hi Team VSCode,

      Just a feature request regarding Python (extension). 

1.       When we write the init(self, blah) method in a class, I want the attributes of the class (self.blah) to be automatically detected using the given parameters from the init function. A small request but might be more time-saving for people like me.
2.       When I used Java, when creating object of a class (maybe imported or user-defined), I automatically get the suggestion to make the name of the object, being the first letter in lower case and rest like that of the class name. I want such feature in Python. I really liked that feature in the RedHat Java extension (thank you Team RedHat) and I would like the same in Python too.

Extension version: 2021.12.0
VS Code version: Code 1.61.2 (6cba118ac49a1b88332f312a8f67186f7f3c1643, 2021-10-19T14:58:13.605Z)
OS version: Linux x64 5.7.0-kali1-amd64
Restricted Mode: No

**Comment by judej** [comment]
Moving this issue to discussion as an enhancement request for comments and upvotes.
